controllers/module_reconciler.go

```diff
@@ -479,3 +496,19 @@ func isNodeSchedulable(node *v1.Node) bool {
 	}
 	return true
 }
+
+func isModuleBuildAndSignCapable(mod *kmmv1beta1.Module) (bool, bool) {
+	buildCapable := mod.Spec.ModuleLoader.Container.Build != nil
+	signCapable := mod.Spec.ModuleLoader.Container.Sign != nil
+	if !buildCapable || !signCapable {
```

Comment:
```suggestion

	if buildCapable && signCapable {
	    return true, true
	}
```

Reply:
No need to iterate if we got all we need already.

Reply:
nit

Reply:
fixed